frontend/src/assets/chevron-left.tsx

```diff
@@ -0,0 +1,28 @@
+interface ChevronLeftProps {
```

Comment:
seems strange to me to put components into the `assets` dir, no?

maybe `components/icons/`?

Reply:
Yes, we probably should: #4541